sub-packages/bionemo-llm/src/bionemo/llm/data/collate.py

```diff
@@ -78,7 +78,7 @@ def bert_padding_collate_fn(
         "text": padding_value,
         "types": 0,
         "attention_mask": False,
-        "labels": -1,
+        "labels": -100,
```

Comment:
whoops 😄 . We should fix this, currently labels are padded with both -1 and -100